src/api-service/__app__/timer_retention/readme.md

```diff
@@ -0,0 +1,11 @@
+# TimerTrigger - Python

```

Comment:
This looks like a generic README.  Perhaps instead of the template documentation, you can include your intent for the function?

Reply:
Added by mistake!